src/main/java/www/purple/mixxy/models/Comic.java

```diff
@@ -0,0 +1,53 @@
+package www.purple.mixxy.models;
+
+/**
+ * Created by Brian_Sabz on 4/5/16.
+ * 
+ * @author Brian_Sabz
+ */
+import com.googlecode.objectify.Ref;
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import com.googlecode.objectify.annotation.Index;
+import com.googlecode.objectify.annotation.Load;
+
+import java.util.Date;
+import java.util.List;
+
+@Entity
+@Index
+public class Comic {
+
+	@Id
+	public Long id;
+
+	// can refer to these Entities using the already fetched Comic, this solve
+	// performance problems
+	// See: https://github.com/objectify/objectify/wiki/Entities#load
+	@Load
+	public Ref<User> author;
+	@Load
+	public Ref<Comic> parent;
+
+	@Index
+	public String title;
+	public String description;
```

Comment:
In the future we may want to use the Search API, or maybe even come up with some clever way of encoding this data so that we can easily search for it ourselves.